src/continuum_absorption/absorption_ff_metal.jl

```diff
@@ -0,0 +1,458 @@
+                   0.109 0.196 0.346 0.545 0.819 1.251]
+
+    LinearInterpolation((T_vals, σ_vals), table_vals, extrapolation_bc=Throw())
+end
+
+_Si_I_ff(ν::Real, T::Real, ndens_SiII::Real, nₑ::Real) =
+    metal_ff_absorption_departure(ν, T, 1, ndens_SiII, nₑ, _departure_term_Si_I_ff)
+
+
+const _departure_term_Mg_I_ff = begin
+    # this comes from table III of Peach 1970 for neutral Magnesium
+    # Warning: it's plausible that the OCR software made transcription errors
+
+    # σ denotes the energy of the photon (in units of RydbergH*Zeff², where Zeff is the net charge
+    # of the species participating in the interaction
+    σ_vals = [0.05, 0.10, 0.15, 0.20, 0.25, 0.30]
+
+    # The temperature (in K)
+    T_vals = [4000, 5000, 6000, 7000, 8000, 9000, 10000, 11000, 12000, 13000, 14000, 15000, 16000,
+              17000, 18000, 19000, 20000, 21000, 22000, 23000, 24000, 25000, 26000, 27000, 28000,
+              29000, 30000, 32000, 34000]
+
+    table_vals = [-0.070  0.008  0.121  0.221  0.274  0.356;
+                  -0.067  0.003  0.104  0.105  0.244  0.325;
+                  -0.066 -0.002  0.091  0.175  0.221  0.302;
+                  -0.065 -0.007  0.080  0.157  0.201  0.282;
+                  -0.065 -0.012  0.069  0.141  0.183  0.264;
+                  -0.065 -0.016  0.059  0.126  0.166  0.248;
+                  -0.065 -0.020  0.049  0.113  0.151  0.232;
+                  -0.066 -0.024  0.040  0.100  0.137  0.218;
+                  -0.066 -0.028  0.032  0.088  0.124  0.205;
+                  -0.066 -0.032  0.025  0.077  0.112  0.194;
+                  -0.066 -0.035  0.018  0.067  0.101  0.183;
+                  -0.066 -0.037  0.012  0.058  0.091  0.173;
+                  -0.066 -0.040  0.006  0.049  0.082  0.164;
+                  -0.066 -0.042  0.001  0.042  0.074  0.157;
+                  -0.066 -0.044 -0.004  0.036  0.067  0.150;
+                  -0.065 -0.045 -0.007  0.030  0.061  0.144;
+                  -0.064 -0.046 -0.011  0.025  0.056  0.139;
+                  -0.063 -0.047 -0.014  0.020  0.051  0.135;
+                  -0.062 -0.048 -0.016  0.017  0.048  0.131;
+                  -0.061 -0.048 -0.018  0.014  0.045  0.128;
+                  -0.059 -0.047 -0.019  0.011  0.042  0.126;
+                  -0.057 -0.047 -0.020  0.009  0.040  0.124;
+                  -0.055 -0.046 -0.020  0.008  0.039  0.123;
+                  -0.053 -0.045 -0.021  0.007  0.038  0.123;
+                  -0.051 -0.044 -0.020  0.006  0.038  0.123;
+                  -0.048 -0.042 -0.020  0.006  0.038  0.123;
+                  -0.045 -0.040 -0.019  0.006  0.039  0.124;
+                  -0.039 -0.035 -0.016  0.008  0.042  0.128;
+                  -0.032 -0.030 -0.012  0.011  0.046  0.133]
+
+    LinearInterpolation((T_vals, σ_vals), table_vals, extrapolation_bc=Throw())
+end
+
+_Mg_I_ff(ν::Real, T::Real, ndens_MgII::Real, nₑ::Real) =
+    metal_ff_absorption_departure(ν, T, 1, ndens_MgII, nₑ, _departure_term_Mg_I_ff)
```

Comment:
I'm a bit skeptical of the utility of these functions.  Is it just about wrapping globals wherever possible?